content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+### DATABASE_URL
+
+`DATABASE_URL` is used to connect to your database. The url format can differ depending on the database it is connecting to.
+Prisma currently supports 4 types of database connectors, of which 3 use the `DATABASE_URL` environment variable. (SQLite sources a file path)
+
+- PostgreSQL
+- MySQL
+- SQLite
+- SQLServer
+
+Example using a PostgreSQL connector:
+
+```js file=.env
+DATABASE_URL = 'postgresql://USER:PASSWORD@HOST:PORT/DATABASE'
+```
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the `Prisma Client`.
+
+Example setting `Prisma Client` level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` strips colors from error messages. It's a truthy value.
+
+```js
+if (process.env.NO_COLOR) {
+  // Do Something
+}
+```
+
+### APPDATA
+
+`APPDATA` is used for accessing the windows root cache directory.
+
+```js
+if (process.env.APPDATA) {
+    return path.join(process.env.APPDATA, 'Prisma')
+}
+```
+
+### INIT_CWD
+
+`INIT_CWD` is the directory where `npm run` was executed. `Prisma` uses it to check for the `schema.prisma` file.
+
+```js
+if (process.env.INIT_CWD) {
+    // Do something
+}
+```
```

Comment:
Not sure we need to document these, they are sort of behind the scene vars that users don't need to know / customize.

Reply:
Removed